src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const shortTermDisc = 20;
+  const longTermDisc = 50;
+  const noDisc = price * days;
+
+  if (days >= shortTerm && days < longTerm) {
+    return noDisc - shortTermDisc;
```

Comment:
The condition `days < longTerm` is unnecessary because if `days` is greater than or equal to `longTerm`, the function would have already returned on line 19. You can remove this condition to simplify the logic.